### PR TITLE
Rename master branch to main

### LINKED ISSUE #118: Rename master to main
@bravo-kernel I'd like to rename `master` to `main` to keep up with the new naming standards. I know that I need to change it in Netlify, and there are bunch of mentions in the repo as well. 

Is there anything else that I don't see?

### PR DESCRIPTION
**Pull request by bravo-kernel**
Closes #118

### CODE REVIEW

**Comment by bravo-kernel** [comment]
@nohwnd I think we are good to go if we follow this order:

1. Merge this PR (because it does break core things, only the edit button etc.)
2. Rename Github branch
3. Update Netlify

**Comment by bravo-kernel** [comment]
@nohwnd feel free to execute

File: docs/additional-resources/articles.table.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const articles = [
     title: "Writing Pester Tests",
     author: "PowerShell Community",
     date: "2016-11-07",
-    url: "https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md",
+    url: "https://github.com/PowerShell/PowerShell/blob/main/docs/testing-guidelines/WritingPesterTests.md",
```
Comment (fflaten) [inline docs/additional-resources/articles.table.js:71]:
```suggestion
    url: "https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md",
```

They still use master 🙂

Comment (bravo-kernel) [reply docs/additional-resources/articles.table.js]: Ohw, totally missed that and totally missed that `Commit Suggestion` button as well. Never seen it before but that's amazing. Thanks 👍

Comment (fflaten) [reply docs/additional-resources/articles.table.js]: No worries. Lightning quick response as usual 😄

**Comment by bravo-kernel** [comment]
Is mr. @nohwnd ok?

**Comment by nohwnd** [comment]
He is dead.

**Comment by nohwnd** [comment]
Merged, renamed, updated.

**Pull request merged**